argus/action_manager/windows.py

```diff
@@ -142,12 +142,13 @@ def install_cbinit(self, service_type):
             # for whatever reason. In this case, we're falling back
             # to use a scheduled task.
             LOG.debug("Cannot install, deploying using a scheduled task.")
-            self._deploy_using_scheduled_task(installer, service_type)
+            self._deploy_using_scheduled_task(installer, service_type,
+                                              activation)
```

Comment:
The `_deploy_using_scheduled_task` method doesn't have the `activation` argument.